spatialos-sdk/examples/project-example/lib/opt.rs

```diff
@@ -0,0 +1,38 @@
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "project-example",
+    about = "A SpatialOS worker written in Rust."
+)]
+pub struct Opt {
+    #[structopt(name = "WORKER_TYPE", long = "worker-type", short = "w")]
+    pub worker_type: String,
+
+    #[structopt(name = "WORKER_ID", long = "worker-id", short = "i")]
+    pub worker_id: Option<String>,
+
+    #[structopt(name = "POLLING_CONNECTION", long = "polling-connection", short = "p")]
+    pub connect_with_poll: bool,
+
+    #[structopt(subcommand)]
+    pub command: Command,
+}
+
+#[derive(Debug, StructOpt)]
+pub enum Command {
+    #[structopt(name = "receptionist")]
+    Receptionist {
+        host: Option<String>,
+        port: Option<u16>,
```

Comment:
There's a missing flag here for `connect_with_external_ip`. This allows you to connect workers from your local machine to cloud deployments along with `spatial cloud connect external`!

Reply:
I added the argument, but verify that I'm using it correctly. I haven't looked much at cloud deployments yet, so I'm not quite sure I'm doing it right.

Reply:
Yup - that's exactly right!